spec/features/user_edits_information_spec.rb

```diff
@@ -0,0 +1,80 @@
+require 'rails_helper'
+include Warden::Test::Helpers
+
+feature "user edits account settings", %(
```

Comment:
I would again add a scenario to make sure that you can't navigate to a 'different' user's settings page.